personal_assistant/pa_note/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+
+
+class Note(models.Model):
+    name = models.CharField(max_length=50, null=False)
+    description = models.CharField(max_length=150, null=False)
```

Comment:
Опис необовʼязковий:
```suggestion
    description = models.CharField(max_length=150, null=True)
```